packages/api-page-builder/src/types.ts

```diff
@@ -812,3 +813,36 @@ export interface BlockCategoryStorageOperations {
     update(params: BlockCategoryStorageOperationsUpdateParams): Promise<BlockCategory>;
     delete(params: BlockCategoryStorageOperationsDeleteParams): Promise<BlockCategory>;
 }
+
+/**
+ * @category RecordModel
+ */
+export interface Block {
+    id: string;
+    name: string;
+    type: "element" | "block";
```

Comment:
Do we really need this? 🤔

Reply:
yes, reply above (used in page builder app - https://github.com/webiny/webiny-js/blob/809fcc5f4a4795d41e69d7fd1b07541e55408046/packages/app-page-builder/src/admin/views/Pages/Editor.tsx#L69)